evennia/objects/objects.py

```diff
@@ -2277,7 +2285,7 @@ class DefaultRoom(DefaultObject):
     This is the base room object. It's just like any Object except its
     location is always `None`.
     """
-
+    _content_types = ("room", "object")
```

Comment:
Add a comment to explain what this is for.

Reply:
How come this is both a "room" _and_ an "object" when "exits" and "characters" are not?

Reply:
What happens if you have a room inside another room? The default return_appearance doesn't have a case for it.

Reply:
A room is, by default, the outermost container possible. In this way, one of the definitions of a room is that it has location `None`. But the typeclass is still Room, and an object, so it will appear in the hosting room as such and you will be able to look at it etc.

Reply:
Right, which is why it is considered an 'object' content_type, so that if it IS somehow inside another room, you can see it. :)

Reply:
It feels somewhat ... strange to handle it like this; you do filtering by content_type with the specific intent of filtering out things of a certain type. Having Rooms not _actually_ abide by that feels wrong.

Reply:
Fair enough. I'll make it just 'room' then. If people stick rooms inside other rooms, they'll have to figure out how to handle that. Or not handle it.